testsuite/tests/prometheus/zync/test_annotations.py

```diff
@@ -10,6 +10,7 @@
 
 pytestmark = [
     pytest.mark.sandbag,  # requires openshift
+    pytest.mark.nopersistence,  # fixture saves pod name, which changes with pod redeployment
```

Comment:
Shouldn't it just save something that matches the pod basename - IMHO there is always some base

Reply:
If you are testing different pod, it is not a persistence test, you are checking the new annotations on a new pod, which is covered by a normal run instead.